canonical-web-and-design/ubuntu.com
# Copy change on /gcp

## Done

Changed 6 to 4.5 and added "an average"

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check and confirm correct changes based on the [copy doc ](https://docs.google.com/document/d/1Gvt9tNFDZkqkBcP0k79C6ueTJCmNZvwoXQoU19WEa8E/edit?ts=6079d63f#)

## Issue / Card
